Taskfile.yml

```diff
@@ -45,7 +45,6 @@ tasks:
       - goreleaser check
       - typos . .github .vscode
       - nixpkgs-fmt --check ./*.nix
-      - staticcheck ./...
```

Comment:
https://github.com/kachick/go-tools/pull/1 may fix this problem, but ... in the first place, I want only faster tools as go vet...

Reply:
https://github.com/golang/go/blob/go1.22.0/src/cmd/vet/README#L1-L33 😍